packages/coachmark/stories/coachmark.stories.ts

```diff
@@ -11,26 +11,308 @@ governing permissions and limitations under the License.
+                total-steps="8"
+                primary-cta="Next"
+                secondary-cta="Previous"
+                .modifierKeys=${modifierKeys}
+                .content=${{
+                    title: 'I am a Coachmark with keys',
+                    description:
+                        'This is a Coachmark with nothing but text in it. Kind of lonely in here',
+                }}
+            >
+                <sp-action-menu placement="bottom-end" quiet slot="actions">
+                    <sp-menu-item>Skip tour</sp-menu-item>
+                    <sp-menu-item>Restart tour</sp-menu-item>
+                </sp-action-menu>
+            </sp-coachmark>
+        </sp-overlay>
+    `;
 };
 
-export const Default = (): TemplateResult => {
+export const withShortCut = (props: Properties): TemplateResult => {
+    const { open = true } = props;
     return html`
-        <sp-coachmark></sp-coachmark>
-        <sp-coachmark variant="dark"></sp-coachmark>
-        <sp-coachmark variant="light"></sp-coachmark>
+        <sp-coach-indicator id="trigger"></sp-coach-indicator>
+        <sp-overlay
+            trigger="trigger@hover"
+            placement="right"
+            .receivesFocus=${'false'}
+            ?open=${open}
+        >
+            <sp-coachmark
+                ?open=${open}
+                primary-cta="Ok"
+                shortcut-key="L"
+                .content=${{
+                    title: 'Coachmark with Shortcut',
+                    description:
+                        'This is a Coachmark with nothing but text in it. Kind of lonely in here',
+                }}
+            ></sp-coachmark>
+        </sp-overlay>
     `;
 };
 
-export const quiet = (): TemplateResult => {
+export const actionButtons = (
+    props: Properties,
+    args: StoryArgs = {}
+): TemplateResult => {
+    const { open = true } = props;
     return html`
-        <sp-coachmark quiet></sp-coachmark>
-        <sp-coachmark quiet variant="dark"></sp-coachmark>
-        <sp-coachmark quiet variant="light"></sp-coachmark>
+        <sp-coach-indicator id="trigger"></sp-coach-indicator>
+        <sp-overlay
+            trigger="trigger@hover"
+            placement="right"
+            .receivesFocus=${'false'}
+            ?open=${open}
+        >
+            <sp-coachmark
+                ?open=${open}
+                primary-cta="Next"
+                secondary-cta="Previous"
+                shortcut-key="L"
+                .content=${{
+                    title: 'Coachmark with Shortcut',
+                    description:
+                        'This is a Coachmark with nothing but text in it. Kind of lonely in here',
+                }}
+                @primary=${({ target }: Event & { target: HTMLElement }) => {
+                    target.dispatchEvent(
+                        new Event('close', { bubbles: true, composed: true })
+                    );
+                    handlePrimary(args);
```

Comment:
Why not something like this
```suggestion
                    args.onPrimary?.();
```
Instead of the deeper indirection of the helper method?

Reply:
Updated as 
```
@primary=${(event: Event & { target: HTMLElement }) => {
        event.target.dispatchEvent(
            new Event('close', { bubbles: true, composed: true })
        );
        args.onPrimary?.(event);
    }}
    @secondary=${(event: Event & { target: HTMLElement }) => {
        event.target.dispatchEvent(
            new Event('close', { bubbles: true, composed: true })
        );
        args.onSecondary?.(event);
    }}
                
```